doc/src/site/sphinx/install/install_and_start.rst

```diff
@@ -6,7 +6,7 @@ This section describes how to download and run Sparkling Water in different envi
 Download and Run Locally
 ------------------------
 
-This section describes how to quickly get started with Sparkling Water.
+This section describes how to quickly get started with Sparkling Water on your personal computer.
```

Comment:
What about: This section describes how to quickly get started with Sparkling Water on your personal computer(Spark's `local` cluster mode) ?